loadtest/tests.go

```diff
@@ -344,10 +412,14 @@ var standardUserEntity UserEntity = UserEntity{
 			Item:   actionDisconnectWebsocket,
 			Weight: 2,
 		},
+		{
```

Comment:
I doubt this is a realistic weight. Can you remove it from here and we can add it back later with a realistic weight?